src/Getintouch/getintouch.module.css

```diff
@@ -0,0 +1,149 @@
+@font-face {
+    font-family: MetaProBold;
+    src: url("./../assets/fonts/MetaPro-Font/MetaPro-Bold.otf") format("opentype");
+  }
+
+  @font-face {
+    font-family: MetaProRegular;
+    src: url("./../assets/fonts/MetaPro-Font/MetaPro-Normal.otf") format("opentype");
+  }
+
+.container{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: row;
+    box-sizing: border-box;
+    width: 100%;
+    background-color: #ffffff;
+    
+}
+
+.formContainer{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    box-sizing: border-box;
+    width:46%;
+    padding: 4rem 0rem 3rem 4rem;
+}
+
+form{
+    width:100%;
+}
+.message{
+    height: 7rem;
+}
+
+label{
+    font-family: MetaProBold;
+    font-size: 1.2rem;
+    line-height: 1.6rem;
+    color: #1F2430 !important; 
+}
+
+.button{
+    background: #4892C7;
+    box-shadow: 0px 4px 10px rgba(72, 146, 199, 0.1);
+    border: none;
+    color: #FFFFFF;
+    font-size: 1.5rem;
+}
+
+input{
+    height:3.5rem;
+    width:90%;
+    margin-bottom: 2rem;
+    border: 1px solid rgba(31, 36, 48, 0.6);
+    border-radius: 5px;
+    padding:0 2rem;
+}
+
+::placeholder{
+    font-family: MetaProRegular;
+    font-size: 1rem;
+    line-height: 1.5rem;
+    color: rgba(31, 36, 48, 0.6);
+}
+
+.textContainer{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    box-sizing: border-box;
+    width:50%;
+}
+
+.heading{
+    display: flex;
+    justify-content: center;
+    align-items: flex-start;
+    flex-direction: column;
+    box-sizing: border-box;
+    font-family: MetaProBold;
+    font-size: 4rem;
+    line-height: 97.8%;
+    color: #1F2430;
+    width: 60%;
+  
+}
+
+.subheading{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    box-sizing: border-box;
+    font-family: MetaProRegular;
+    font-size: 1.5rem;
+    line-height: 2rem; 
+    width:60%;
+}
+
+@media screen and (max-width: 583px){
+
+.container{
+    flex-direction: column;
+}
+
+.textContainer{
+    width:100%;
+    margin-bottom: 3rem;
+}
+
+.formContainer{
+    width:100%;
+    padding: 2rem 3rem;
+}
+
+input{
+    width:100%;
+}
+
+.heading{
+    align-items: center;
+    width: 80%;
+  
+}
+
+.subheading{
+    text-align: center;
+    line-height: 1.8rem; 
+    width:90%;
+}
+
+}
+
+@media screen and (max-width: 365px){
```

Comment:
Are these the preffered media query break points?

Reply:
I was just decreasing the font size so that it will look good even on the smallest phone screen.